### ISSUE TITLE
ansible-galaxy install --force no longer replaces roles

### ISSUE THREAD

**Issue by jxn** [ansible/galaxy-issues#249]
After the version bump to 2.3.0, it seems install --force no longer forces an overwrite.  We have been using this to "upgrade" roles stored in git, but they are no longer pulled in.

Here's the command used:
`ansible-galaxy install --force -r ansible/requirements.yml -p ansible/roles/`

Here's an example requirements.yml
```
---
- src: git@git.srv:ansible/linux-http.git
  scm: git
- src: git@git.srv:ansible/apache-php.git
  scm: git
```

Previously, when we pushed new role updates to master, these would be overwritten in the 'ansible/roles' directory with the above command.  Now, these are not updated until we manually delete the roles in that folder.   We get the following output:
```
- linux-http is already installed, skipping.
- apache-php is already installed, skipping.
```
As far as I can tell, force no longer does anything as of 2.3.0

**Comment by chouseknecht** [comment]
Fixed in by [Ansible PR #23391](https://github.com/ansible/ansible/pull/23391)

**Comment by chouseknecht** [comment]
This issue was moved to ansible/galaxy#74
